packages/blade/src/components/Tag/AnimatedTag.native.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import type { EasingFn } from 'react-native-reanimated';
+import Animated, {
+  Easing,
+  useSharedValue,
+  withTiming,
+  useAnimatedStyle,
+  runOnJS,
+  Keyframe,
+} from 'react-native-reanimated';
+import type { ReanimatedKeyframe } from 'react-native-reanimated/lib/typescript/reanimated2/layoutReanimation/animationBuilder/Keyframe';
```

Comment:
Lets just do `typeof Keyframe`. Should be the same thing

Reply:
I tried this but getting this error for some reason 😕 
<img width="680" alt="image" src="https://github.com/razorpay/blade/assets/30949385/1baedb67-093d-4db7-a7ef-2bfaa5074617">